offchain-modules/packages/x/src/handlers/ckb.ts

```diff
@@ -743,6 +815,75 @@ export class CkbHandler {
     }
   }
 
+  async parseUnlockTx(tx: Transaction): Promise<NervosUnlockAssetTxMetaData | null> {
```

Comment:
The right `parseUnlockTx` logic:

1. verify: tx input[0] should be committeeMultisigCkbCell or committeeMultisigSudtCell
2. verify: witnesses.length === input.length + 1 && can parse unlockMemo from witnesses[input.length] succeed
3. get info: get burnIds from witness
4. get info: get unlock info from outputs[0...burnIds.length], the burnId and outputCell are one-to-one

Besides, the asset of unlockedCells should  be one kind asset(CKB or one kind sUDT)

Reply:
fixed,
1. inputs[0] === committeeMultisigCkbCell or committeeMultisigSudtCell
2. verify witnesses length, and try parse unlockMemo
3. get burnIds from witness
4. parse output info, verify same kind asset